### PR TITLE
actionlint: Add version 1.5.3

### PR DESCRIPTION
**Pull request by guitarrapc**
:octocat: Static checker for GitHub Actions workflow files.

It is useful for every GitHub Actions users and work well on Windows, too.

### CODE REVIEW

File: bucket/actionlint.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": ":octocat: Static checker for GitHub Actions workflow files",
+    "homepage": "https://github.com/rhysd/actionlint",
+    "version": "1.5.2",
+    "license": "MIT",
+    "url": "https://github.com/rhysd/actionlint/releases/download/v1.5.2/actionlint_1.5.2_windows_amd64.zip",
```
Comment (Ash258) [inline bucket/actionlint.json:6]: Add 32bit version

File: bucket/actionlint.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "hash": "f0b2842f0b3712483951a9607f7e3f06f2e4e0cded0fb3f73f5cb7cc61a6fe46",
+    "bin": "actionlint.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/rhysd/actionlint/releases/download/v$version/actionlint_$version_windows_amd64.zip"
```
Comment (Ash258) [inline bucket/actionlint.json:11]: Add 32bit version

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/actionlint.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (guitarrapc) [reply bucket/actionlint.json]: added 32bit

File: bucket/actionlint.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (guitarrapc) [reply bucket/actionlint.json]: added 32bit

[the remark this answers is not in the document]
Comment (guitarrapc) [reply bucket/actionlint.json]: done.

File: bucket/actionlint.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": ":octocat: Static checker for GitHub Actions workflow files",
```
[the remark this answers is not in the document]
Comment (guitarrapc) [reply bucket/actionlint.json]: done

File: bucket/actionlint.json (hunk 1 above)
Comment (guitarrapc) [reply bucket/actionlint.json]: done

File: bucket/actionlint.json (hunk 2 above)
Comment (guitarrapc) [reply bucket/actionlint.json]: done

**Comment by guitarrapc** [comment]
confirm auto updater successfully detect new version.
https://github.com/guitarrapc/scoop-bucket/commit/46ec68b8156ab157aca3f1d2e43d189303629f54

File: bucket/actionlint.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
+    "description": "Static checker for GitHub Actions workflow files",
+    "homepage": "https://github.com/rhysd/actionlint",
+    "version": "1.5.2",
+    "license": "MIT",
```
Comment (Ash258) [inline bucket/actionlint.json:5]:
```suggestion
    "version": "1.5.2",
    "description": "Static checker for GitHub Actions workflow files",
    "homepage": "https://github.com/rhysd/actionlint",
    "license": "MIT",
```

File: bucket/actionlint.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    "bin": "actionlint.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/rhysd/actionlint/releases/download/v$version/actionlint_$version_windows_amd64.zip",
+                "hash": {
+                    "url": "$baseurl/actionlint_$version_checksums.txt"
+                }
```
Comment (Ash258) [inline bucket/actionlint.json:24]:
```suggestion
                "url": "https://github.com/rhysd/actionlint/releases/download/v$version/actionlint_$version_windows_386.zip"
```

File: bucket/actionlint.json [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+            },
+            "32bit": {
+                "url": "https://github.com/rhysd/actionlint/releases/download/v$version/actionlint_$version_windows_386.zip",
+                "hash": {
+                    "url": "$baseurl/actionlint_$version_checksums.txt"
+                }
```
Comment (Ash258) [inline bucket/actionlint.json:30]:
```suggestion
                "url": "https://github.com/rhysd/actionlint/releases/download/v$version/actionlint_$version_windows_386.zip"
```

File: bucket/actionlint.json [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+            }
+        }
```
Comment (Ash258) [inline bucket/actionlint.json:32]:
```suggestion
        },
        "hash": {
            "url": "$baseurl/actionlint_$version_checksums.txt"
        }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/actionlint.json (hunk 5 above)
Comment (guitarrapc) [reply bucket/actionlint.json:24]: windows_386? just remove trailing `,`?

Comment (guitarrapc) [reply bucket/actionlint.json]: done

**Comment by guitarrapc** [comment]
@Ash258 Thank you for the review, all fixes commited.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**